apps/sms/views/conversation/js/conversation.js

```diff
@@ -129,15 +133,27 @@ var ConversationView = {
                               this.onVisibilityChange);
 
     this.toField.addEventListener(
-      'keypress', this.toFieldKeypress.bind(this), true
+      'keypress', 
+      Utils.throttle(this.toFieldKeypress.bind(this),
```

Comment:
And another really minor reminder is the message app's convention: if you want to make the some parameters into another new line, please set all the parameter into new line with 2 space indention.